### PR TITLE
[QOLSVC-7965] add scripts to assist in querying the database from batch servers

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

File: files/default/psql-env-ckan.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ "$CKAN_PROPERTY" = "" ]; then
+  CKAN_PROPERTY="sqlalchemy[.]url"
```
Comment (duttonw) [inline files/default/psql-env-ckan.sh:4]: what about datastore user?

File: files/default/psql-env-ckan.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+fi
+CONNECTION_STRING=$(grep "$CKAN_PROPERTY" /etc/ckan/default/production.ini |awk -F '//' '{print $2}')
+export PGUSER=$(echo "$CONNECTION_STRING" | awk -F ':' '{print $1}')
```
Comment (duttonw) [inline files/default/psql-env-ckan.sh:7]: and what about dba root user but this still fills in the pghost

**Review by duttonw — APPROVED** [review]

File: files/default/psql-env-ckan.sh (hunk 2 above)
Comment (ThrawnCA) [reply files/default/psql-env-ckan.sh:7]: Batch servers, by design, do not have access to the DBA credentials. If you want to perform DBA operations, you need to go to SSM Parameter Store and look up the name and password yourself. At that point you're probably implementing a change request that should have details like the host prefilled anyway.

File: files/default/psql-env-ckan.sh (hunk 1 above)
Comment (ThrawnCA) [reply files/default/psql-env-ckan.sh:4]: Just need to provide the right property to look up.

**Pull request merged**